index.html

```diff
@@ -2153,18 +2153,9 @@ <h2 id="OfflineAudioContext">
               <li>
                 <p>
                   Let <dfn>[[rendered buffer]]</dfn> be an internal slot
-                  initialized with an <a>AudioBuffer</a> constructed by
-                </p>
-                <pre>
-new AudioBuffer({
-    numberOfChannels: contextOptions.numberOfChannels,
-    length: contextOptions.length,
-    sampleRate: contextOptions.sampleRate
-})
-                </pre>
-                <p>
-                  This buffer holds the rendered audio data for the offline
-                  context.
+                  initialized to <code>undefined</code>. When rendering begins,
+                  this slot will hold an <a>AudioBuffer</a> containing rendered
+                  audio data for the offline context.
```

Comment:
I'm not sure (since I didn't add the text regarding the slot), but I imagined that the reason it exists is to supply the buffer on the control thread side when the Promise succeeds. So I just left it as-is.  Let's discuss this today on the call but I don't have any particular leaning; happy to do whatever works best.